drag_and_drop_v2/public/css/drag_and_drop.css

```diff
@@ -472,6 +473,21 @@
   font-size: 0.875em;
 }
 
+.xblock--drag-and-drop .btn-link {
+    padding: 1px;
+    font-size: 0.875em;
+}
+
+.xblock--drag-and-drop .btn-default.btn-link:hover,
```

Comment:
Why is it necessary to handle the focus border explicitly? What is preventing the default behavior from just working?